core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -765,7 +761,7 @@ class GroupCoordinator(val brokerId: Int,
 
   def tryCompleteJoin(group: GroupMetadata, forceComplete: () => Boolean) = {
     group.inLock {
-      if (group.notYetRejoinedMembers.isEmpty)
+      if (!group.hasNotYetRejoinedMembers)
```

Comment:
nit: since this is the only use of `hasNotYetRejoinedMembers`, maybe we can change it to `hasAllMembersJoined` and remove the negation?

Reply:
done